include/ardpolyfills/Arduino.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
don't we need to add the file name here?

Reply:
Solved in 095e0795